src/common/Constants.cpp

```diff
@@ -85,6 +85,17 @@ ReadVerifierExclusionListFromConstantsFile() {
   return result;
 }
 
+const vector<pair<uint64_t, uint32_t>>
+ReadVerifierMicroblockExclusionListFromConstantsFile() {
+  auto pt = PTree::GetInstance();
+  vector<pair<uint64_t, uint32_t>> result;
+  for (auto& entry : pt.get_child("node.verifier.microblock_exclusion_list")) {
```

Comment:
I would prefer to specify const auto& here.
Also, what does entry.second stand for? I think it'd be more readable to have:
const auto& <...> = entry.second;

Reply:
Perfect.